package.json

```diff
@@ -1,118 +1,14 @@
 {
-  "name": "@sentry/wizard",
-  "version": "3.5.0",
-  "homepage": "https://github.com/getsentry/sentry-wizard",
-  "repository": "https://github.com/getsentry/sentry-wizard",
-  "description": "Sentry wizard helping you to configure your project",
-  "keywords": [
-    "sentry",
-    "wizard",
-    "sdk",
-    "cli",
-    "project",
-    "setup",
-    "install",
-    "configure"
+  "version": 3,
```

Comment:
I think this file should not be changed...? 😅

Reply:
lol nope 😅